Makefile

```diff
@@ -1,2 +1,26 @@
-all:
-	g++ project.cpp -o project -lsfml-graphics -lsfml-window -lsfml-system -lsfml-audio
\ No newline at end of file
+CXX = g++
+CXXFLAGS = -D _DEBUG -D _SHOW_STACK_TRACE -ggdb3 -std=c++17 -O0 -Wall -g
```

Comment:
Move debug flags to separate variable, like DEBUG_FLAGS e.t.c (Compile stable version of project with CXXFLAGS = -O2 -Wall)